tests/unit/matrix/test_measure.py

```diff
@@ -1290,4 +1337,49 @@ def it_computes_its_blocks_to_help(self, request, dimensions_):
 
     @pytest.fixture
     def dimensions_(self, request):
-        return (instance_mock(request, Dimension), instance_mock(request, Dimension))
+        return instance_mock(request, Dimension), instance_mock(request, Dimension)
+
+
+class Describe_Zscores(object):
+    """Unit test suite for `cr.cube.matrix.measure._Zscores` object."""
+
+    def it_computes_zscore_subtotals_blocks_to_help(self, request, dimensions_):
+        weighted_cube_counts_ = instance_mock(request, _BaseWeightedCubeCounts)
+        _weighted_cube_counts = property_mock(
+            request, _BaseSecondOrderMeasure, "_weighted_cube_counts"
+        )
+        _weighted_cube_counts.return_value = weighted_cube_counts_
```

Comment:
1. This property mock should be on `_Zscores`, not its base class.
2. The return value can be specified in the `property_mock()` call as a keyword parameter `return_value=weighted_cube_counts_.
3. The name should be `_weighted_cube_counts_prop_`, however no name will be needed when you move the return value into the mock call and it can just start out as `property_mock(...`.